compile.mk

```diff
@@ -38,7 +38,7 @@ TEST_FLAG_PARAM := -DTEST_FLAG=$(TEST_FLAG)
 else
 TEST_FLAG_PARAM :=
 endif
-COMPILE_CMD=ant -f scripts$(D)build_test.xml -DTEST_ROOT=$(TEST_ROOT) -DBUILD_ROOT=$(BUILD_ROOT) -DJDK_VERSION=$(JDK_VERSION) -DJDK_IMPL=$(JDK_IMPL) -DJDK_VENDOR=$(JDK_VENDOR) -DJCL_VERSION=$(JCL_VERSION) -DBUILD_LIST=${COMPILE_BUILD_LIST} -DRESOURCES_DIR=${RESOURCES_DIR} -DSPEC=${SPEC} -DTEST_JDK_HOME=${TEST_JDK_HOME} -DJVM_VERSION=$(JVM_VERSION) -DLIB_DIR=$(LIB_DIR) ${TEST_FLAG_PARAM}
+COMPILE_CMD=ant -f scripts$(D)build_test.xml -DTEST_ROOT="$(TEST_ROOT)" -DBUILD_ROOT="$(BUILD_ROOT)" -DJDK_VERSION="$(JDK_VERSION)" -DJDK_IMPL="$(JDK_IMPL)" -DJDK_VENDOR="$(JDK_VENDOR)" -DJCL_VERSION="$(JCL_VERSION)" -DBUILD_LIST="${COMPILE_BUILD_LIST}" -DRESOURCES_DIR="${RESOURCES_DIR}" -DSPEC="${SPEC}" -DTEST_JDK_HOME="${TEST_JDK_HOME}" -DJVM_VERSION="$(JVM_VERSION)" -DLIB_DIR="$(LIB_DIR)" ${TEST_FLAG_PARAM}
```

Comment:
Could we only quote JDK_VENDOR or non directory one unless this can be tested in windows platforms (jenkins or local)?

Reply:
Also, we may need to use `$(Q)` to replace `"` and test it works on all platforms.